pkg/apis/lbaas/v2/lb_cluster_types.go

```diff
@@ -0,0 +1,26 @@
+package v2
+
+import (
+	"go.anx.io/go-anxcloud/pkg/apis/common/gs"
+)
+
+type LoadBalancerImplementation string
+
+const (
+	LoadBalancerImplementationHAProxy LoadBalancerImplementation = "haproxy"
+)
+
+// anxcloud:object
+
+// Cluster holds the information of a load balancing cluster
+type Cluster struct {
+	gs.GenericService
+	gs.HasState
+
+	Identifier       string                     `json:"identifier,omitempty" anxcloud:"identifier"`
+	Name             string                     `json:"name,omitempty"`
+	Implementation   LoadBalancerImplementation `json:"implementation,omitempty"`
+	FrontendPrefixes *gs.PartialResourceList    `json:"frontend_prefixes,omitempty"`
+	BackendPrefixes  *gs.PartialResourceList    `json:"backend_prefixes,omitempty"`
```

Comment:
https://github.com/anexia-it/go-anxcloud/pull/309#discussion_r1379886562

It might make sense to both omit and clear the backend prefixes.
Frontend prefixes are always required, but for consistency, we decided to keep it as is.